### ISSUE TITLE
File extension problem

### ISSUE THREAD

**Issue by noembryo** [c3er/mdview#57]
These are the app's specs (it would be nice if copying these, produced a markdown text too.. just saying 😃 )
Windows 10 22H2
```
{
    "applicationName": "Markdown Viewer",
    "applicationDescription": "Standalone Markdown Viewer",
    "applicationVersion": "3.1.1",
    "homepage": "https://github.com/c3er/mdview",
    "issueLink": "https://github.com/c3er/mdview/issues",
    "frameworkVersions": [
        [
            "electron",
            "30.0.3"
        ],
        [
            "chrome",
            "124.0.6367.119"
        ],
        [
            "node",
            "20.11.1"
        ],
        [
            "v8",
            "12.4.254.15-electron.0"
        ]
    ]
}
```

The application opens the files that have an extension of `.MD` as simple texts.
Just renaming the extension fixes this, but I don't see the point in this, since Windows filenames are not case sensitive..

Anyway, thank you for this nice app.. 🙏

**Comment by c3er** [comment]
Thank you for filing this issue!

> The application opens the files that have an extension of `.MD` as simple texts.
> Just renaming the extension fixes this, but I don't see the point in this, since Windows filenames are not case sensitive..

You're right that Windows doesn't make a difference but strings in (virtually) any programming language are case sensitive if no extra step was made. But you're right, that it doesn't make sense to distinguish between `.md` and `.MD` (in any OS). I'll release a fixed version hopefully soon.

Until then, there is another workaround: in the settings dialog is a checkbox "Render files of this type as Markdown"

![Screenshot of settings dialog](https://github.com/user-attachments/assets/d7e289bc-1dab-497f-b6eb-5d20b209b9a8)

You can reach the settings dialog via "Edit" --> "Settings" or be pressing Ctrl+, (Control + comma).

> These are the app's specs (it would be nice if copying these, produced a markdown text too.. just saying 😃 )

Please make another issue, if you really wish this 🙂

> Anyway, thank you for this nice app.. 🙏

You're welcome! Thank you!

**Comment by noembryo** [comment]
> > These are the app's specs (it would be nice if copying these, produced a markdown text too.. just saying 😃 )
> 
> Please make another issue, if you really wish this 🙂

Nah, it's not something that I really need very often (hopefully 😉).
It's just something that I would do if I wanted for my programs to report their info to the world.
Fortunately, for now, I just let the users send me error logs, so a simple text is enough for that..

Anyway, I'll close that now since it will be fixed in a future update..
🙏

**Comment by c3er** [comment]
I released [version 3.2.0](https://github.com/c3er/mdview/releases/tag/v3.2.0) which compares the file extensions now case insensitive.
